src/dmd/globals.d

```diff
@@ -219,6 +294,7 @@ struct Param
 
     bool run; // run resulting executable
     Strings runargs; // arguments for executable
+    Array!EnvironmentVariable envToRestoreBeforeRun;
```

Comment:
This is only accessed by private functions, and shouldn't be in global params accessible by all.